lib/SILOptimizer/Differentiation/PullbackCloner.cpp

```diff
@@ -2052,6 +2052,8 @@ bool PullbackCloner::Implementation::run() {
   SmallVector<SILValue, 8> retElts;
   // This vector will contain all indirect parameter adjoint buffers.
   SmallVector<SILValue, 4> indParamAdjoints;
+  // This vector will identify the locations where initialization is needed.
+  SmallVector<bool, 8> outputsToInitialize;
```

Comment:
Is it possible to use a `SmallBitVector` instead?

Reply:
Thanks for the suggestion, I've gone ahead and done so.